### PR TITLE
Try to be clear with sub-clock/base-clock.

### PR DESCRIPTION
**Pull request by HansOlsson**
@henrikt-ma - I can add this if you think it clarifies.

### CODE REVIEW

File: chapters/synchronous.tex [hunk 1]
```diff
@@ -405,7 +405,7 @@ \section{Clock Constructors}\label{clock-constructors}
 The optional $\mathit{startInterval}$ argument (defaults to 0) is the value returned by \lstinline!interval()! at the first tick of the clock, see \cref{initialization-of-clocked-partitions}.
 The result is of base type \lstinline!Clock! that ticks when \lstinline!edge($\mathit{condition}$)! becomes \lstinline!true!.
 \begin{nonnormative}
-This clock is used to trigger a clocked partition due to a state event, that is a zero-crossing of a \lstinline!Real! variable, in a continuous-time partition or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
+This clock is used to trigger a clocked base-clock partition due to a state event, that is a zero-crossing of a \lstinline!Real! variable, in a continuous-time base-clock partition or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
```
Comment (henrikt-ma) [inline chapters/synchronous.tex]: The placement of commas makes this sentence hard to parse (not really the topic of this PR, but it makes it easier to accept the new text):
```suggestion
This clock is used to trigger a clocked base-clock partition due to a state event (that is a zero-crossing of a \lstinline!Real! variable) in a continuous-time base-clock partition, or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
```

File: chapters/synchronous.tex [hunk 2]
```diff
@@ -443,7 +443,7 @@ \section{Clock Constructors}\label{clock-constructors}
 \end{operatordefinition*}
 
 Besides inferred clocks and solver clocks, one of the following mutually
-exclusive associations of clocks are possible in one base partition:
+exclusive associations of clocks are possible in one base-clock partition:
```
Comment (henrikt-ma) [inline chapters/synchronous.tex]: Time to go sentence-based, but for some reason I can't make a Github suggestion.

File: chapters/synchronous.tex [hunk 3]
```diff
@@ -1204,7 +1204,7 @@ \subsection{Solver Methods}\label{solver-methods}
 The simulation environment might also combine all partitions associated with method \lstinline!"External"!, as well as all continuous-time partitions, and integrate them together with the solver selected by the simulation environment.
 \end{nonnormative}
 
-If the solver method is \emph{not} \lstinline!"External"!, then the partition is
+If the solver method is \emph{not} \lstinline!"External"!, then the sub-clock partition is
 integrated using the given method with the step-size \lstinline!interval()!.
```
Comment (henrikt-ma) [inline chapters/synchronous.tex]: Time to go sentence-based:
```suggestion
If the solver method is \emph{not} \lstinline!"External"!, then the sub-clock partition is integrated using the given method with the step-size \lstinline!interval()!.
```

File: chapters/synchronous.tex [hunk 4]
```diff
@@ -1213,7 +1213,7 @@ \subsection{Solver Methods}\label{solver-methods}
 
 The solvers are defined with respect to the underlying ordinary
 differential equation in state space form to which the continuous-time
-partition can be transformed, at least conceptually ($t$ is time,
+base-clock partition can be transformed, at least conceptually ($t$ is time,
```
Comment (henrikt-ma) [inline chapters/synchronous.tex]: Time to go sentence-based.

Comment (henrikt-ma) [reply chapters/synchronous.tex]: By the way, shouldn't this be _sub-clock_?
```suggestion
sub-clock partition can be transformed, at least conceptually ($t$ is time,
```

**Review by henrikt-ma — COMMENTED** [review]
I think being explicit like this is good, but I'd like to hear what the language group has to say about the very verbose terminology being used in so many places.

At least when _clocked_ or _unclocked_ is present, wouldn't the terms _(un)clocked base-partition_ and _clocked sub-partition_ be nicer and just a clear?

File: chapters/synchronous.tex (hunk 4 above)
Comment (HansOlsson) [reply chapters/synchronous.tex]: Yes, I was confused by "continuous-time partition"; as we only have "continuous-time base-clock partition" but this is a description of the transformation of the "conceptual non-discretized" version of the _discretized sub-clock partition_.
